Changes

```diff
@@ -219,6 +219,9 @@ Working version
   modules with mismatching -for-pack
   (Pierre Chambart and Vincent Laviron, review by Mark Shinwell)
 
+- #11635: Turn warning 31 (Module_linked_twice) into a hard error
```

Comment:
Please list the issues as well, not just the PR.

Reply:
Done